### PR TITLE
Fix broken reference to DeadLetterQueue*Manager

### PR DESCRIPTION
**Pull request by robbavey**
Change test to refer to DeadLetterQueueWriter instead of DeadLetterQueueWriteManager
Remove unnecessary (and broken) reference to DeadLetterQueueReadManager
Change variable and access to refer to ...Reader, instead of ...ReadManager

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 
 ## 1.0.1
   - internal: rename DeadLetterQueueManager to DeadLetterQueueReader
+
+## 1.0.2
```
Comment (talevy) [inline CHANGELOG.md]: we usually place the versions in the reverse order

example: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/CHANGELOG.md

**Review by talevy — CHANGES_REQUESTED** [review]
left a quick note regarding the CHANGELOG. LGTM otherwise

**Comment by suyograo** [comment]
@robbavey should we merge this, since you're fixing the tests in #4 ?

**Comment by robbavey** [comment]
Sure, I think I can go ahead - Tal's objection was pretty cosmetic

**Review by talevy — APPROVED** [review]

**Pull request merged**